cppcache/src/ThinClientPoolHADM.cpp

```diff
@@ -292,11 +293,11 @@ void ThinClientPoolHADM::removeCallbackConnection(TcrEndpoint* ep) {
   m_redundancyManager->removeCallbackConnection(ep);
 }
 
-void ThinClientPoolHADM::sendNotConMesToAllregions() {
+void ThinClientPoolHADM::sendNotConnectedMessageToAllRegions() {
   std::lock_guard<decltype(m_regionsLock)> guard(m_regionsLock);
   for (std::list<ThinClientRegion*>::iterator it = m_regions.begin();
```

Comment:
Can this be changed to a for range loop?

Reply:
Yep, fixed.